grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java

```diff
@@ -162,10 +162,11 @@ public Object newClient(ClientBuilderParams params) {
         } else {
             jsonMarshaller = null;
         }
+        final boolean useMethodMarshaller = options.get(GrpcClientOptions.USE_METHOD_MARSHALLER);
```

Comment:
Let's do this in the constructor of `ArmeriaChannel`, similar to how we handle `UNSAFE_WRAP_RESPONSE_BUFFERS`.

Reply:
Applied feedbacks, Thanks.